metatron/src/main/java/uk/gov/ida/eidas/metatron/domain/EidasCountryConfig.java

```diff
@@ -62,6 +70,22 @@ public void setEnabled(boolean enabled) {
         this.enabled = enabled;
     }
 
+    public void setMetadataTruststore(KeyStore metadataTruststore) {
```

Comment:
The setters in here are never used, and probably never will be as the values are being deserialized from the country config. Do we need them at all?